exporter/clickhouseexporter/config.go

```diff
@@ -151,11 +150,7 @@ func (cfg *Config) buildDB(database string) (*sql.DB, error) {
 
 // ShouldCreateSchema returns true if the exporter should run the DDL for creating database/tables.
 func (cfg *Config) ShouldCreateSchema() bool {
```

Comment:
There is no reason to have these methods exported. All all them should be private.

Reply:
Submitted https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33647

Reply:
Then, this one can be actually removed

Reply:
Agreed, we can merge this and then update the other PR?

Reply:
They can be merged independently